docs/apache-airflow/best-practices.rst

```diff
@@ -135,26 +135,21 @@ Bad example:
       catchup=False,
       tags=["example"],
   ) as dag:
-
+      @task()
       def print_array():
```

Comment:
The example should show calling `print_array()` as well. Otherwise the task _technically_ doesn't get added to the DAG. Any users, I suspect especially beginners, could copy/paste this code and not understand why the `print_array` task doesn't appear in the DAG.

Reply:
Thanks, I had a feeling someone might mention this.